artifactory-enterprise/azuredeploy.json

```diff
@@ -0,0 +1,865 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "type": "string",
+      "defaultValue": "Standard_A4",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "clusterName": {
+      "type": "string",
+      "metadata": {
+        "description": "String used as a base for naming resources. Must be 3-61 characters in length and globally unique across Azure. A hash is prepended to this string for some resources, and resource-specific information is appended."
+      },
+      "maxLength": 61
+    },
+    "nodeCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "minValue": 1,
+      "metadata": {
+        "description": "Number of VM instances (100 or less) for member artifactory nodes. Minimum 1 node."
+      },
+      "maxValue": 100
+    },
+    "artifactoryVersion": {
+      "type": "string",
+      "defaultValue": "5.9.0",
```

Comment:
should we have allowed values here?

Reply:
Yes. It's required. I use it for installing Artifactory.